mlir/lib/Conversion/MemRefToEmitC/MemRefToEmitCPass.cpp

```diff
@@ -50,6 +53,34 @@ struct ConvertMemRefToEmitCPass
     if (failed(applyPartialConversion(getOperation(), target,
                                       std::move(patterns))))
       return signalPassFailure();
+
+    mlir::ModuleOp module = getOperation();
+    module.walk([&](mlir::emitc::CallOpaqueOp callOp) {
+      if (callOp.getCallee() != "aligned_alloc" &&
```

Comment:
the free strings aren't declared somewhere? (i.e. instead of using "aligned_alloc" or "malloc" or, below "cstdlib" or "stdlib.h", you'd use some `const char*` declared somewhere)

Reply:
in fact, I recommend doing that, cstdlib and stdlib.h appear in 2 places and this way you'd avoid accidental typos.